.github/workflows/automation-ci.yml

```diff
@@ -0,0 +1,171 @@
+#TODO:
+# - use variables to make somethings dry, like path-names, routes.
+# - Caching web and express build dependencies for faster builds.
+# - better way to include the fhir-web-automation repo since its private
+
+name: Run automation tests
+on:
+  workflow_dispatch:
+    inputs:
+      branch: 
+        required: true
+        description: The branch to target
+
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+      # setup code, directories and system wide dependency
+      - name: Checkout web
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+          path: web
+          fetch-depth: 1
+
+      - name: Checkout express server
+        uses: actions/checkout@v3
+        with:
+          repository: onaio/express-server
+          path: express
+
+      - name: Checkout fhir automation
+        uses: actions/checkout@v3
+        with:
+          repository: onaio/fhir-web-automation
+          token: ${{ secrets.ACCESS_TOKEN }} # currently using a PAT -> not the best option.
+          path: fhir-web-automation
+          ref: p-netm-patch-1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      # package, library installation and app configuration.
+
+      - name: Provision web envs
+        run: |
+          tee -a web/app/.env \n<<EOF
+          # fhir
+          SKIP_PREFLIGHT_CHECK=true
```

Comment:
Is there a way we can reuse the sample env file here?

Reply:
yes, but we would still need to override several values, this gives us full control in how the app is configured, we shall know any configs supplied to the app were fed in from here.